src/main/java/io/pakland/mdas/githubstats/application/mappers/TeamMapper.java

```diff
@@ -5,6 +5,6 @@
 
 public final class TeamMapper {
     public static Team dtoToEntity(TeamDTO teamDTO) {
-        return Team.builder().id(teamDTO.getId().longValue()).name(teamDTO.getName()).slug(teamDTO.getSlug()).build();
+        return Team.builder().id(teamDTO.getId()).slug(teamDTO.getSlug()).build();
```

Comment:
The `name` prop is not needed (just asking)? I can't remember now...

Reply:
> The `name` prop is not needed (just asking)? I can't remember now...

Oh, you're right, the Name was not the same as the slug I tink I got confused when reading the liquidbase script and i thought the slug and name field were the same and deleted the second one. Should we keep working with the name? i think the main use of the name or the slug would be the information shown from each team on the final .csv report.